modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -228,10 +229,28 @@ export default class MVTLayer extends TileLayer {
     }
   }
 
-  _onViewportLoad() {
-    super._onViewportLoad();
-    this._onViewportChange();
+  _setWGS84PropertyForTiles() {
```

Comment:
- Shall we move this method to a pure function? Passing `selectedTiles`from `updateState` like:
`setWGS84PropertyForTiles(tileset.selectedTiles);`
- `propName` can be outside the function then?